controllers/managedocs_controller.go

```diff
@@ -436,8 +433,16 @@ func (r *ManagedOCSReconciler) reconcilePhases() (reconcile.Result, error) {
 			r.reconcileStrategy = v1.ReconcileStrategyNone
 		}
 
-		if err := r.get(r.addonParamSecret); err != nil {
-			return ctrl.Result{}, fmt.Errorf("Failed to get the addon param secret, Secret Name: %v", r.AddonParamSecretName)
+		// Read the secret and store it in addonParams map
+		addonParamSecret := &corev1.Secret{}
+		addonParamSecret.Name = r.AddonParamSecretName
+		addonParamSecret.Namespace = r.namespace
+		if err := r.get(addonParamSecret); err != nil {
+			return ctrl.Result{}, fmt.Errorf("Failed to get the addon parameters secret %v", r.AddonParamSecretName)
+		}
+		for key, value := range addonParamSecret.Data {
+			fmt.Println(key, value)
```

Comment:
- I believe, it isn't good to either print or log contents of addonParamSecret which may contain onboarding ticket and storage endpoints
- @nb-ohad your take on this?

Reply:
I missed that, yes that should be removed. Thanks for pointing it out.

Reply:
The onboarding ticket is a one-time use token, so by the time it appears in the log it was already consumed.
But your point remains, especially around user-provided emails.

@rewantsoni What was the intention of printing the secret data into the log at this location?
Usually when we have an error regarding a specific parameter we print the specific parameter with that error, if the parameter is not to be disclosed in a log then we just do not print it

Reply:
I have updated the PR without the print statement